### PR TITLE
[CI] Fix creating list of artifacts for CodeCov.

### PR DESCRIPTION
**Pull request by mtl1979**
* Remove trailing comma from list of CodeCov artifacts to avoid warning and subsequent error state.

### CODE REVIEW

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -674,7 +674,7 @@ jobs:
       shell: bash
       run: |
         python3 -u -m pip install --user gcovr==5.0
-        python3 -m gcovr -j 3 --verbose \
+        python3 -m gcovr ${{ matrix.build-dir || '' }} -j 3 --verbose \
```
Comment (nmoinvaz) [inline .github/workflows/cmake.yml:677]: Why is this needed? I thought that is the purpose of the `--root` arg below.

Comment (mtl1979) [reply .github/workflows/cmake.yml:677]: For out-of-source builds, the build/binary directory needs to be sub-directory of the source directory for `--root` argument to work, it's limitation of filtering in `gcovr`... However it is possible to pass binary directory separately without that restriction.

**Review by Dead2 — APPROVED** [review]

File: .github/workflows/cmake.yml (hunk 1 above)
Comment (nmoinvaz) [reply .github/workflows/cmake.yml:677]: Why was out of source builds previously working fine?

Comment (mtl1979) [reply .github/workflows/cmake.yml:677]: I don't know why but recently I noticed that the size of coverity artifact dropped from few tens of kilobytes on normal builds to about 300 bytes on OSB builds. It was pretty clear hint that no coverity data was found or collected.
